### DISCUSSION TITLE
First person for RE4?

### DISCUSSION THREAD

**Discussion by brianx87** [praydog/REFramework#938] [General Questions]
Can we expect to see first person mode for RE4?

RE2 and RE3 were incredible in first person on the Steam Deck, I'm really hoping the same for 4, the mods just don't cut it like REFramework would.

**Comment by amanzano33** [comment]


**Reply by 909990999900** [reply]
I've tried that mod but it's just not the same without being able to toggle it on and off.... the reframe work first person is just too smooth. I really hope we get a re4 version

**Reply by Rayjaxx79** [reply]
Is it possible for anyone to fork their own version while Praydog is away perhaps?

**Comment by Draverd** [comment]
Yeah could we please get a first person VR Mod? We would greatly appreciate it.

Post Script: My health has been improving, and at some point I plan on recording myself playing a bunch of VR Games God willing. That includes using this Mod for Resident Evil 2 & 3. If we can somehow get a first person Mod before June 2024 that would be incredible. If we can get the Mod sooner like by March or April it would make me incredibly happy. Even if somehow I am still not recording yet I could at least play the Game both for fun, and to train my Marksmanship.

Also thanks amanzano33 that is better than nothing

**Comment by Satan-friend** [comment]
The question is who will be first. Praydog with its mod or Capcom with its DRM that will block it :(
I'm starting to regret buying RE 4 with the add-on. Without Praydog's mod I have no desire to play it :/

**Comment by 909990999900** [comment]
Where is first person mode?? The framework was just updated but there's no first person option? Why can't we just get this, every other game has it?!

**Reply by Satan-friend** [reply]
I guess it's all about the new kicking action or something. Also probably about Capcom's recent hostile attitude towards mods.

Anyway, I hope it will come one day. Can't wait motion controls to RE4. I will never buy a Resident Evil game again before Praydog/Capcom release VR :D

**Comment by Blackstarreach** [comment]
Please praydog, if you are reading this I would do anything for RE4 first person! RE2 is so much fun in first person

**Comment by Satan-friend** [comment]
Me too. Maybe some paid patronite or whatever to play? I'd be happy to pay for access to RE 4 on VR in first person.

**Comment by sacer11** [comment]
Oh yes please !!!
I really enjoyed playing resident evil 7,8 and 2 and 3 again in VR - really nice experience, except the end of RE3, where I am missing a cursor to shoot Nicolai in the final scene, even in third person view.
It was real cool experience with WiFi 6e, 4080 and Quest3
I really was in the game :)
So hope to see forward to UEVR and of cause, Resident Evil 4 in first person with oculus Touch controllers - please don’t let us wait too long ….

**Comment by Satan-friend** [comment]
I know that these questions can be flustering. I admire your work and patience. But can you tell us anything about the first person for RE4? :)

Just whether it's in the plans. I don't know if it's better to wait or play the original on Quest (I'd rather go through the remake on VR first).

**Comment by armchairjedi** [comment]
First, THANK YOU, Capcom owes you bigtime.  I bought the entire RE series because of your mods. They are so much more fun to play in VR.  too bad RE5 is now pretty much unmoddable.   RE4 first person PLEASE, I am sure number of us would support you on Patreon.  UEVR rocks.  Motion controls allow you to play games standing up and get some exercise vs sitting in a chair with an xbox controller.

**Comment by armchairjedi** [comment]
in the meantime, VRified games has some workarounds to create the first person experience in RE4 with motion controls https://www.youtube.com/watch?v=ZYdy7Gk_xgw&ab_channel=VRifiedGames

**Reply by Satan-friend** [reply]
It looks really bad. It's a shame that Praydog has abandoned the RE 4 project; there are still test footage on his channel.

**Reply by Fredhoob** [reply]
I don't think he has abandoned it, he apparently has a lot of other projects going on at the moment and RE4 isn't a priority anymore.

**Reply by Satan-friend** [reply]
He is now focusing on UEVR and DD 2. I sincerely doubt that he will return to the project he started a year ago. Let's hope I'm wrong.

**Reply by sacer11** [reply]
hope he will find time to integrate a first person, RE4 would be one of the most excelent games for Quest 3

**Reply by jossm87** [reply]
It could be awesome. Wish one day it will arrive...

**Reply by Kentypoo** [reply]
I bought and beat  RE2, RE3, RE7 and RE8 because of Praydogs first person motion controlled VR mods. They are all top 10 vr games in my opinion. I messed around with UEVR a lot when it first came out but would rather have a polished motion controlled first person RE4 VR mod and haven't  bought RE4 remake yet because I wait and Pray Dog!😉 I feel like RE4 is the only reason to drop $1000 on a ps5/psvr2 but would rather play it like I did RE 2,3,7,8, in my 2 car garage with my Wireless Vive Pro Eye and my RTX4090 using OVRAS's manual redirected walking. Love you Praydog, hope you can complete this someday soon!

**Reply by 909990999900** [reply]
Is there any update to this? The games been out for a year and a half. Where's the first person support ????

**Reply by HairyHog** [reply]
I don't understand why this was abandoned. I don't even need the VR version, I just want to play it in first person without the buggy FPS mod from Nexus that's been in beta for ages

**Comment by KOSKing09** [comment]
Is praydog ever going to respond to this or is he even still modding?. I've been waiting patiently since it came out for the first person mod I wont play the re4 without it i beat all the others.... now its just like im stuck on this one waiting.... lmfao

**Comment by ProCodec** [comment]
Check out [praydog/REFramework] When is resident evil 4 going to support
first person VR in this mod? (Issue #1128) Tldr: praydog said it's not
planned

**Reply by HairyHog** [reply]
very weird response from him, why we don't just get some official explanation i don't know

**Comment by 909990999900** [comment]
Hey praydog can you release first person for re4 already. We saw it working on Twitter when the game launched and it was said that you had a few kinks to iron out. What the hell is going on? Communicate with us at the very least. the whole reason I bought re4 on pc was to play it in first person same way as I did re2 and 3.

**Comment by Draverd** [comment]
praydog if money is the issue, since it will cost you time and effort. I am sure everyone here, if not most of us, would be willing to pitch in to fund this. I see you have a Patreon with 1,419 Total members, and 396 Paid members. Well you could create a paid tier specifically for this. Then we can help support you so that you can develop this.

**Comment by MuhammadJA87** [comment]
After seeing the first person view for RE2R and RE3R, I was impressed of how polished it is. It even has its own FoV and camera settings which don't affect the main cutscenes.

I really hope you can make one for RE4R. Even if all the other animations turn to third person for a moment like the kicking or getting grabbed then switch back to first person is good enough.

Even capcom's own VR version for the PS5 turn to third person when performing melee. Not even capcom can make a melee attack in VR. Or maybe they can but too lazy.

**Comment by MuhammadJA87** [comment]
So now RE9 is confirmed to have both first and third person view, is there any hope we'll get to see first person for RE4?

**Reply by KOSKing09** [reply]
we are a still waiting i still havent played RE4 because the first person mod isnt out.

**Reply by Satan-friend** [reply]
Since Praydog dropped first-person support despite the announcement and advanced status of work on RE4, I doubt we'll ever get support for RE9 or other RE games 😭

Maybe Capcom will put the VR DLC up for sale? I hope that...

**Reply by MuhammadJA87** [reply]
If he just will give us the beta build that he showed in his YT channel, it's good enough. At least it's better than the one in Nexusmods where you have to install the headless mod to play it.

**Reply by MuhammadJA87** [reply]
https://www.youtube.com/watch?v=D6s2rmGrQro

Someone in the comments in this video is saying "The first person stable movement mod released on praydog"

Not sure if serious or troll.

**Comment by Draverd** [comment]
Praydog do we need to set up a GoFundMe for $500, or a thousand, ten thousand, etc... to get you to finally to do this? What will it take?

**Comment by MuhammadJA87** [comment]
There's a new first person mod for the game if anyone is interested.

https://www.nexusmods.com/residentevil42023/mods/5458

**Reply by 909990999900** [reply]
> There's a new first person mod for the game if anyone is interested.
> 
> 
> 
> https://www.nexusmods.com/residentevil42023/mods/5458

Omg you fucking saint!!! Thanks so much for this, I haven't played in over a year and I never would have known this existed. 

Super happy this guy did what praydog refused to do! What a legend 

And it's toggleable in game just like praydogs for RE2 and RE3 absolute chefs kiss

**Comment by Draverd** [comment]
EVERYONE!!!! There is now a way to play the game on PCVR with full motion controls!!!!!!!!!!!! Fyi I would periodically search for a solution to our problem. So roughly every month, or so give and take. Well I just came across this video, https://www.youtube.com/watch?v=TkLagX-wrLQ

Now they suggest installing some additional things, in the video, but I was able to get everything to work with these three mods:

RE4VR
https://www.nexusmods.com/residentevil42023/mods/5682

Fluffy Mod Manager
https://www.nexusmods.com/residentevil42023/mods/14

REFramework
https://www.nexusmods.com/residentevil42023/mods/12

Now I don't know what your specs are, but I was able to get it up, and running instantly without a hitch.


My Processor:	AMD Ryzen 7 5800X 8-Core Processor, 4200 Mhz, 8 Core(s), 16 Logical Processor(s)

I am using a  BaseBoard Product:	TUF B450M-PLUS GAMING

32 Gigs of DDR4 Ram

And, lastly I have an RTX 3070 TI for a Graphics card


I share this information so that you can kind of have a base line you know? So adjust accordingly. Whether that be through settings, or potentially additional hardware.


Oh yeah one last thing. The VR headset that I am using is an HTC VIVE COSMOS: ELITE. Now it is not the latest wireless Elite, but the previous wired iteration. I understand that depending upon the headset a person can have their performance potentially tank. That is with the higher end headsets. So remember to factor everything in. Cheers, and have fun!
  



EDIT: If you are deciding to upgrade your setup it would costs you $1,200 minimum to buy just the things that I mentioned for my PC. Not counting a case, fan, power supply, storage, etc... I have six Terabytes of storage, and 500 Gigabytes our of that is SSD.

Also, you can buy the Cosmos Elite Headset for $549.99 on average. That is only the headset mind you, and not Base Stations, or controllers. I owned the original HTC VIVE so I still have the Base Stations from that, but my original two controllers slowly fell apart, and I had to replace those a few years ago.   

So with just some of the things I mentioned you are looking at around $1,749.99


https://www.amazon.com/AMD-Ryzen-5800X-16-Thread-Processor/dp/B0815XFSGK?tag=ustxtaddt-20&th=1

https://www.amazon.com/TUF-Gaming-Motherboard-Ryzen-B450M-Plus/dp/B07FKV3TVF

https://www.ebay.com/p/6037510118

https://www.amazon.com/ddr4-ram/s?k=ddr4+ram&tag=ustxtaddt-20

https://www.amazon.com/NVIDIA-GeForce-Express-Graphics-Platinum/dp/B097PZT7J3?tag=ustxtaddt-20


I share all of this to give everyone a general idea for general costs. That is with more of a higher end of a medium setup at this time.
